docs-source/spring/data/menu/main.yaml

```diff
@@ -17,9 +17,9 @@ main:
     sub:
       - name: Spring Operations Center
         ref : "/platform/soc"
-      - name: API Gateway
+      - name: Apache APISIX Gateway
```

Comment:
i've tried to avoid long names so the menu does not overly busy and renders nicely, we've used generic terms here rather than product names